predict.py

```diff
@@ -1,30 +1,20 @@
 # Load libraries
-import flask
-import pandas as pd
+import pickle
 import tensorflow as tf
-from tensorflow import keras
-from keras import layers
+from keras.layers import TextVectorization
 from keras.models import load_model
 
 from embedding import PositionalEmbedding
 from transform import TransformerEncoder
 
-from keras.layers import TextVectorization
```

Comment:
@aritradey97 can you please add it as a different file? If we need to run this again and again, we will loose this file.

Reply:
I have changed the default predict.py to predict_func.py.